modules/openapi-generator/src/main/resources/go-experimental/api_doc.mustache

```diff
@@ -15,6 +15,7 @@ Method | HTTP request | Description
 ## {{{operationId}}}
 
 > {{#returnType}}{{{returnType}}} {{/returnType}}{{{operationId}}}(ctx, {{#allParams}}{{#required}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/required}}{{/allParams}}{{#hasOptionalParams}}optional{{/hasOptionalParams}})
+
```

Comment:
@zippolyte can you please make the same change to https://github.com/OpenAPITools/openapi-generator/blob/2844149fbac5b39843e04502224f532d6750abc6/modules/openapi-generator/src/main/resources/go/api_doc.mustache#L17 ?